src/Mouse Position/mousePosition.js

```diff
@@ -0,0 +1,109 @@
+L.Control.AstroMousePosition = L.Control.extend({
+  options: {
+    separator: " : ",
+    numDigits: 5,
+    prefix: "",
+    targetPlanet: ""
+  },
+
+  onAdd(map) {
+    this.container = L.DomUtil.create("div", "leaflet-control-mouseposition");
+    L.DomEvent.disableClickPropagation(this.container);
+    map.on("mousemove", this.onMouseMove, this);
+
+    this.map = map;
+
+    this.lonTo180 = false;
+    this.planetocentric = true;
+    this.postiveEast = true;
+
+    this.astroMath = new AstroMath(this.options.targetPlanet);
```

Comment:
Does it make sense to have targetPlanet as a parameter since it is needed for this to work instead of having it as an option?

Reply:
Nevermind. We can add a method to AstroMap to get the target name.